src/pages/instant-observability.js

```diff
@@ -498,28 +499,35 @@ const QuickstartsPage = ({ data, location }) => {
           <div
             css={css`
               display: grid;
-              grid-gap: 1rem;
+              grid-gap: 1.25rem;
               grid-template-columns: repeat(4, 1fr);
-              grid-auto-rows: minmax(var(--guide-list-row-height, 150px), auto);
+              grid-auto-rows: minmax(
+                var(--guide-list-row-height, 150px),
+                287px
+              );
```

Comment:
We might want to double-check this is working as intended. After discussing it, I don't think we want these to have variable heights.